config/environments/development.rb

```diff
@@ -59,4 +59,6 @@
     Bullet.bullet_logger = true
     Bullet.add_footer = true
   end
+
+  config.hosts = nil
```

Comment:
Why not configuring `ENV['APP_HOST']`?

Reply:
I mean sure for test and prod. But why would we need a host limitation in dev?

Reply:
So that the evil hackerman can't spoil your development system